src/website/interface.ts

```diff
@@ -165,8 +165,14 @@ export class WebsiteInterface {
 
 		const players = await this.api.getPlayers(tournamentId);
```

Comment:
Can you convert this into a Map where the keys are the discordId? This adds more `.find` calls, which each take linear time but are usually called in a loop context.